### ISSUE TITLE
Dotted spaces on macos

### ISSUE THREAD

**Issue by 4ntoine** [BertrandBev/code_field#12]
Using:
> code_text_field: ^1.0.1-2

Having the following source code:
```
return new CodeField(
      controller: this._controller,
      textStyle: TextStyle(fontFamily: 'SourceCode'),
      lineNumberStyle: LineNumberStyle(),
      expands: true,
    );
```

It looks good in browser:
<img width="266" alt="Снимок экрана 2021-08-25 в 18 32 15" src="https://user-images.githubusercontent.com/846671/130799949-90394f48-2469-4996-91f9-63917663b736.png">

but on macos the spaces are dotted:
<img width="267" alt="Снимок экрана 2021-08-25 в 18 33 36" src="https://user-images.githubusercontent.com/846671/130800075-d7623179-e7bd-4b9f-9ca3-1c571babf72f.png">

Is it code_field issue? Any workaround?
I'm happy to assist if necessary.

**Comment by 4ntoine** [comment]
BTW i can see the dots on android too (so it seems to be not macos but font issue):
![Screenshot_1629899361](https://user-images.githubusercontent.com/846671/130802839-461396c7-57aa-4455-bba9-521eeb26b513.png)

**Comment by johannphilippe** [comment]
Same on Linux here.

**Comment by johannphilippe** [comment]
It seems related to the `_spaceToMiddleDots` method. I could fix it by removing calls to this method and  the equivalent code at the end of `set value` setter. Of course, I don't know what this method is used for, but it might be fixed more accurately for cross platforms by not removing the function.

**Comment by 4ntoine** [comment]
@johannphilippe i can't find any method `_spaceToMiddleDots ` in this repo, you can please explain how you solved/workarounded it?

**Comment by BertrandBev** [comment]
My bad, I forgot one line in my previous commit. Fixed it in `650e6fb88`

**Comment by 4ntoine** [comment]
I can confirm it's fixed in "1.0.2". Thanks!

**Comment by mikhalchenko-alexander** [comment]
I have dots issue when I copy (ctrl+c / crtl+v) code from web to any other editor like IDE or Sublime Text. Is this the same issue?
